substrate/frame/asset-conversion/src/types.rs

```diff
@@ -186,3 +159,99 @@ impl<AssetId: Ord + Clone> MultiAssetIdConverter<NativeOrAssetId<AssetId>, Asset
 		}
 	}
 }
+
+/// Credit of [Config::Currency].
+///
+/// Implies a negative imbalance in the system that can be placed into an account or alter the total
+/// supply.
+pub type NativeCredit<T> =
+	CreditFungible<<T as frame_system::Config>::AccountId, <T as Config>::Currency>;
+
+/// Credit (aka negative imbalance) of [Config::Assets].
+///
+/// Implies a negative imbalance in the system that can be placed into an account or alter the total
+/// supply.
+pub type AssetCredit<T> =
+	CreditFungibles<<T as frame_system::Config>::AccountId, <T as Config>::Assets>;
+
+/// Credit that can be either [`NativeCredit`] or [`AssetCredit`].
+///
+/// Implies a negative imbalance in the system that can be placed into an account or alter the total
+/// supply.
+#[derive(RuntimeDebug, Eq, PartialEq)]
+pub enum Credit<T: Config> {
```

Comment:
this type will be dropped when this issue addressed - https://github.com/paritytech/polkadot-sdk/issues/1842